src/opm/output/eclipse/Summary.cpp

```diff
@@ -3502,22 +3503,29 @@ namespace Evaluator {
 
     std::string Factory::functionUnitString() const
     {
+        const auto unit_string_tracer = this->es_.tracer()
+            .get_unit_string(this->es_.getUnits(),
+                             this->node_->keyword);
+
+        if (unit_string_tracer != "") {
```

Comment:
.empty()

Reply:
> .empty()

Good point.  I've pushed an update to that effect.